python/ray/serve/_private/replica_result.py

```diff
@@ -62,6 +63,7 @@ def __init__(
         self._is_streaming: bool = metadata.is_streaming
         self._request_id: str = metadata.request_id
         self._object_ref_or_gen_sync_lock = threading.Lock()
+        self.__lazy_object_ref_or_gen_asyncio_lock = None
```

Comment:
Please reserve __ for Python internal names

```suggestion
        self._lazy_object_ref_or_gen_asyncio_lock = None
```